contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html lang = "en">
+	<!-- http://www.w3schools.com/html/html_layout.asp -->
+	
   <title>
   Poodle Security Training School | Contact Us
   </title>
-<body style="font-family:verdana;">
-<!-- http://www.w3schools.com/html/html_layout.asp -->
-<center>
-	<header><h1>Poodle Security Training School</h1></header>
+  <link rel="shortcut icon" href="http://www.hddevelop.com/images/pawprint.ico">
+  <meta charset="utf-8">
+	
+	
+	<style>
+	body{
+	  font-family: Verdana, sans-serif;
+	}
+	
+	.center{
+	  text-align: center;
+	}
+	
+	.bars{
+	  background-color: gray;
+	  color:white;
+	  padding: 25px 0px 25px 0px;
+	}
+	  
+	.content{
+	  padding: 25px 0px;
+	}
+	
+	p{
+	  padding: 0px 25px;
+	}
+  </style>
+	
+	
+<body>
+	<header class="center bars"><h1>Poodle Security Training School</h1></header>
 
-	<nav><a href="index.html">Homepage</a> | <a href="classes.html">Classes</a> | <a href="about.html">About Us</a> | Contact Us</nav>
-</center>
+	<nav class="center" style="padding: 10px 0px 0px 0px"><a href="index.html">Homepage</a> | <a href="classes.html">Classes</a> | <a href="about.html">About Us</a> | Contact Us</nav>
 
+<section class="content">
 <p>
 We are located at 222 Fake St. Cleveland, MS 38732<br>
-Or you can reach us at our email address: vwebster@deltastate.edu
+Or you can reach us at our email address: <a href="mailto:vwebster@deltastate.edu">vwebster@deltastate.edu</a>
 </p>
+</section>
 
-<footer><center>Website by <a href="http://www.hddevelop.com/">HD Develop</a> | Streamlined solutions.</center></footer>
+<footer class="center bars">Website by HD Develop <b>|</b> Streamlined solutions.</footer>
```

Comment:
again, ``<center>`` is no longer supported. added the ``.center`` and ``.bars`` classes.

removed the hyperlink on the "HD Develop" text, because as of now it would go to the Poodle homepage. But in real situations, it would link to our company's website